Player/Player.gd

```diff
@@ -94,7 +128,7 @@ func process_movement(delta: float) -> void:
 	var accel: float = ACCEL if direction.dot(hvel) > 0 else DEACCEL
 
 	if is_on_floor():
-		if Input.is_action_just_pressed("jump"):
+		if Input.is_action_just_pressed("jump") && !devmode:
```

Comment:
The change here is no longer needed since flying code is strictly in the fly() function